__tests__/index.test.js

```diff
@@ -26,6 +26,14 @@ describe('api/categories', () => {
                 expect();
             });
     });
+    test('GET:404 responds with a message of "Wrong URL" when provided with wrong end point', () => {
```

Comment:
This should actually be in its own describe block, whereas here its inside the api/categories endpoint describe block.